sky/serve/serve_utils.py

```diff
@@ -490,9 +490,8 @@ def wait_service_registration(service_name: str, job_id: int) -> str:
                     raise ValueError(
                         f'The service {service_name!r} is already running. '
                         'Please specify a different name for your service. '
-                        'To update an existing service, run: `sky serve down` '
-                        'and then `sky serve up` again (in-place update will '
-                        'be supported in the future).')
+                        'To update an existing service, run: sky serve update '
+                        '<service-name> <new-service-yaml>')
```

Comment:
Is it possible to fill in the `<>` fields to make it easier to copy and run? For example, 
```suggestion
                        'To update an existing service, run: sky serve update '
                        f'{service_name} <new-service-yaml>')
```